gfx/common/win32_common.c

```diff
@@ -2173,13 +2169,21 @@ void win32_set_style(MONITORINFOEX *current_mon, HMONITOR *hm_to_use,
       rect->right     = *width;
       rect->bottom    = *height;
 
+      bool window_show_decor = settings->bools.video_window_show_decorations;
```

Comment:
Hi, can you declare this variable at the top of the code block? This will otherwise create C89 build errors. Thanks.

Reply:
Will do.

Reply:
Done - rebased/squashed.